### ISSUE TITLE
☂ Polls

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-flutter#164]
I.e., this Zulip feature:
https://zulip.com/help/create-a-poll

This is an umbrella issue. The different aspects of poll support are in several sub-issues:
* #165 
* #166 
* #167 
* #168 
* #169 

I expect we won't do these all at the same time. In particular #165 and #166 are features zulip-mobile has and should happen before launch, but the others are features it doesn't (and haven't been near the top of the list of features we regularly hear requests for, either) and can happen later.

**Comment by gnprice** [comment]
Closing this umbrella issue; it's better tracked through the individual sub-issues.

(The remaining three open sub-issues #167, #168, #169 are all now in the "M7: Future" milestone, so first I moved this umbrella issue to that same milestone.)
